### PR TITLE
WebKit export of https://bugs.webkit.org/show_bug.cgi?id=190642

### PR DESCRIPTION
**Pull request by woaiwyhty**
WebKit export from bug: [MediaRecorder should fire a stop event when all tracks are ended](https://bugs.webkit.org/show_bug.cgi?id=190642)

### CODE REVIEW

File: mediacapture-record/MediaRecorder-stop.html [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+            assert_equals(recorder.state, "inactive", "MediaRecorder has been stopped when all tracks are ended");
+            t.done();
+        });
+        assert_equals(video.getVideoTracks().length, 1, "video mediastream starts with one track");
+        recorder.start();
+        assert_equals(recorder.state, "recording", "MediaRecorder has been started successfully");
+        video.getVideoTracks()[0].stop();
+        t.step_timeout(() => {
+            assert_unreached("stop event is not fired after 2 seconds");
+        }, 2000);
```
Comment (Ms2ger) [inline mediacapture-record/MediaRecorder-stop.html]: Generally, we prefer just letting the test time out rather than adding an explicit timeout like this.

**Comment by foolip** [comment]
@woaiwyhty @Ms2ger do you know what happened here? The test changes were landed in WebKit in https://trac.webkit.org/changeset/237311/webkit but this was never merged...

**Comment by foolip** [comment]
Sorry for the spam everyone, https://github.com/web-platform-tests/wpt-pr-bot/issues/38 is the issue about why @wpt-pr-bot does stuff with closed PRs when it shouldn't.

**Comment by foolip** [comment]
Looks like mediacapture-record/MediaRecorder-stop.html (the only file originally in this PR) was later exported in https://github.com/web-platform-tests/wpt/pull/13805. So this should remain closed. Pardon again everyone for the spam. I'll remove the labels once again and then make sure to do nothing at all to avoid triggering @wpt-pr-bot again. This PR should remain closed and no additional action is needed.

**Comment by Ms2ger** [comment]
@foolip I guess it was forgotten; stuff is still super-manual in WebKit, as you know. What did you do, though???

**Comment by foolip** [comment]
Now that https://github.com/web-platform-tests/wpt-pr-bot/issues/38 I can (hopefully) explain what happened here without the bot adding back all the reviewers and comments.

On this PR, a rebase or merge went wrong and lots of files were considered affected. wpt-pr-bot labeled and added reviewers accordingly. The issue of >100 labels was addressed in https://github.com/web-platform-tests/wpt-pr-bot/issues/35, but things could still go wrong like this again.

Anyway, no action is needed here, because the changes were landed in another PR already.

**Pull request closed without merging**